README.md

```diff
@@ -106,19 +91,19 @@ const MyPlaidComponent = () => {
 
 `PlaidLink` wraps the view you provide as a child with a `Pressable` component and intercepts the `onPress` event.
 
-### OAuth requirements
+#### OAuth requirements
 
-For Link Token based OAuth support, you must configure your `link_token` with a `redirect_uri` to support OAuth on iOS. Other than setting the `redirect_uri`, which must be a universal link, when you create the `link_token` no further configuration is required. Notably, no props are required on the React Native side.
+For Link Token based OAuth support, you must configure your `link_token` with a [redirect_uri](https://plaid.com/docs/api/tokens/#link-token-create-request-redirect-uri) to support OAuth on iOS. On Android you need to register your package name as described above. Other than setting the `redirect_uri`, which must be a universal link, when you create the `link_token` no further configuration is required. Notably, no props are required on the React Native side.
 
 For non-Link Token based OAuth support, you must pass two props to the PlaidLink React Native component:
 1. `oauthRedirectUri` this is the same uri you would pass to the `redirect_uri` for Link Token based OAuth. It must be registered as a universal link.
 2. `oauthNonce` this is a 16 character nonce.
```

Comment:
Should we delete this? non-link-token based oauth support is hella deprecated.

Reply:
Good call. I removed this.